src/table/components/head/ColumnAdjuster.tsx

```diff
@@ -16,53 +18,93 @@ const ColumnAdjuster = ({ column, isLastColumn }: AdjusterProps) => {
   const { rootElement, applyColumnWidths, constraints } = useContextSelector(TableContext, (value) => value.baseProps);
   const columnWidths = useContextSelector(TableContext, (value) => value.columnWidths);
   const setColumnWidths = useContextSelector(TableContext, (value) => value.setColumnWidths);
-  const tempWidths = useRef({ columnWidth: 0, initX: 0, initWidth: 0 });
+  const tempWidths = useRef({ adjusterHitArea: {}, columnWidth: 0, initX: 0, initWidth: 0 });
   const borderHeight = rootElement.getBoundingClientRect().height - PAGINATION_HEIGHT + BORDER_WIDTH;
 
   if (!applyColumnWidths || constraints.active) return null;
 
-  const mouseMoveHandler = (evt: MouseEvent) => {
-    const deltaWidth = evt.clientX - tempWidths.current.initX;
-    const adjustedWidth = Math.max(tempWidths.current.initWidth + deltaWidth, MIN_COLUMN_WIDTH);
-
+  const updateWidth = (adjustedWidth: number) => {
     tempWidths.current.columnWidth = adjustedWidth;
     const newColumnWidths = [...columnWidths];
     newColumnWidths[pageColIdx] = adjustedWidth;
     setColumnWidths(newColumnWidths);
   };
 
+  const confirmWidth = () => {
+    if (tempWidths.current.columnWidth !== tempWidths.current.initWidth) {
+      const newWidthData = { type: ColumnWidthTypes.PIXELS, pixels: tempWidths.current.columnWidth };
+      applyColumnWidths(newWidthData, column);
+    }
+  };
+
+  const mouseMoveHandler = (evt: MouseEvent) => {
+    const deltaWidth = evt.clientX - tempWidths.current.initX;
+    const adjustedWidth = Math.max(tempWidths.current.initWidth + deltaWidth, MIN_COLUMN_WIDTH);
+    updateWidth(adjustedWidth);
+  };
+
   const mouseUpHandler = (evt: MouseEvent) => {
-    evt.preventDefault();
-    evt.stopPropagation();
+    preventDefaultBehavior(evt);
     document.removeEventListener('mousemove', mouseMoveHandler);
     document.removeEventListener('mouseup', mouseUpHandler);
 
-    if (tempWidths.current.columnWidth !== tempWidths.current.initWidth) {
-      const newWidthData = { type: ColumnWidthTypes.PIXELS, pixels: tempWidths.current.columnWidth };
-      applyColumnWidths(newWidthData, column);
+    confirmWidth();
+    let adjusterHitArea = tempWidths.current.adjusterHitArea as HTMLDivElement;
+    if (adjusterHitArea.id !== 'adjuster-hit-area') {
+      adjusterHitArea = adjusterHitArea.parentElement as HTMLDivElement;
     }
+    adjusterHitArea.blur();
   };
 
   const mouseDownHandler = (evt: React.MouseEvent) => {
     // evt.preventDefault();
     evt.stopPropagation();
 
     tempWidths.current = {
+      adjusterHitArea: evt.target,
```

Comment:
I would say better to do `evt.target.closest('#adjuster-hit-area')` to remove the logic in `mouseUpHandler`